### ISSUE TITLE
could you share the code？

### ISSUE THREAD

**Issue by sunzhe09** [kirito12138/AFF-Net#1]
I cann't replemention the loss with Itracker dataset。if possible，could you send your code to my email？  my address is 297862023@qq.com

**Comment by kirito12138** [comment]
The loss function is Smooth L1 Loss, which is provided by PyTorch.
You may check [PyTorch doc of SmoothL1Loss](https://pytorch.org/docs/stable/generated/torch.nn.SmoothL1Loss.html?highlight=smooth%20l1#torch.nn.SmoothL1Loss) for details.
Sorry for the delay of code releasing, I will release source codes in this repo in the near future.
